clang/test/Driver/sycl-offload-header-check.cpp

```diff
@@ -0,0 +1,11 @@
+// REQUIRES: system-windows
+// Test the integrated header file name generation.  Name should match the
+// actual path name and not the environment variable setting
+// RUN: mkdir -p %t_DiRnAmE
+// invoke the compiler overriding output temp location
+// RUN: env TMPDIR=%t_dirname TEMP=%t_dirname TMP=%t_dirname  \
+// RUN: %clang_cl -### -fsycl %s 2>&1 | \
```

Comment:
Also to clarify: is `%clang` never invoked on Windows?

Reply:
You can run %clang on Windows.   I will add a test for it.